indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -559,9 +559,10 @@ def _process(self):
                          .all())
         signals.event.designer.print_badge_template.send(self.template, regform=self.regform,
                                                          registrations=registrations)
+        file_name_prefix = 'Tickets' if config_params.pop('is_ticket', False) else 'Badges'
```

Comment:
You could even simplify this:
```suggestion
        file_name_prefix = 'Tickets' if config_params.pop('is_ticket') else 'Badges'
```